src/main/java/org/radarcns/management/web/rest/SubjectResource.java

```diff
@@ -288,6 +290,27 @@
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
+    /**
+     * API to get subjects by date created.
+     *
+     * @param fromDate date range beginning
+     * @param toDate date range ending
+     * @param pageable the page information
+     * @return the requested page of revisions
+     */
+    @GetMapping(value = "/subjects", params = { "fromDate", "toDate" })
+    public ResponseEntity<List<SubjectDTO>> getSubjectsByDateCreated(
+            @RequestParam(value = "fromDate")
+            @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) ZonedDateTime fromDate,
+            @RequestParam(value = "toDate")
+            @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) ZonedDateTime toDate,
+            @PageableDefault(page = 0, size = Integer.MAX_VALUE) Pageable pageable)
+            throws NotAuthorizedException {
+        log.debug("REST request to get subjects");
+        List<SubjectDTO> subjects = subjectService.findByDateCreated(fromDate, toDate);
```

Comment:
This list should definitely be filtered to only include subjects where token.hasPermissionOnSubject is true.